src/app/interfaces/catalog.interface.ts

```diff
@@ -18,6 +18,7 @@ export interface Catalog {
   trains: {
     [trainName: string]: CatalogTrain;
   };
+  error: boolean;
```

Comment:
Is this an API field? If not, please make a separate interface to extend from Catalog.

Reply:
Yeah, it's an API field.